src/Calendar/Calendar.js

```diff
@@ -519,6 +563,7 @@ class Calendar extends Component {
                         data-is-focused={day.isSame(currentDateDisplayed)}
                         key={copyDate}
                         onClick={isEnabledDate(day, this.props) ? () => this.dateClick(copyDate, enableRangeSelection) : null}
+                        onFocus={this.handleDayFocus(day)}
```

Comment:
should this be `onFocus={() => this.handleDayFocus(day)}`?

Reply:
Lint complains about creating a function in a loop. This is why the function declarations you pointed out seem weird as well. Yes, even though the `onClick` looks the same, just the focus one triggered the lint rule.